cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -24,10 +24,11 @@ import (
 	"sort"
 	"strings"
 
-	stringutils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/string"
-	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 	computeBeta "google.golang.org/api/compute/v0.beta"
 	"google.golang.org/api/compute/v1"
+
```

Comment:
Why the separation of imports here?

Reply:
In cli_tools, we've been splitting into three groups: standard library, 3p deps, compute-image-tools; eg: https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e